src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +85,21 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+
+    String getPrintableString(Printable... printables) {
+        String summary= "";
+        for(Printable p: printables){
+            summary+=p.getPrintableString();
+            summary+=" ";
+        }
+        return summary;
+    }
+
```

Comment:
eliminate such unnecessary vertical white spaces, which results in unwanted scrolling.